src/components/App.tsx

```diff
@@ -2909,6 +2993,21 @@ class App extends React.Component<AppProps, AppState> {
         scenePointer.x,
         scenePointer.y,
       );
+
+      if (isEraserActive(this.state)) {
+        const elements = this.scene.getElements().map((ele) => {
+          if (this.elementIdsToErase[ele.id]) {
+            return newElementWith(ele, { isDeleted: true });
+          } else if (hitElement && ele.id === hitElement.id) {
+            return newElementWith(ele, { isDeleted: true });
+          }
+          return ele;
+        });
+        this.updateScene({ elements, commitToHistory: true });
```

Comment:
I don't think we should be using this API internally as it was designed for host app purposes. Right now it doesn't hurt, but long-term we might make it so that `updateScene` doesn't notify host apps etc.